odoo/tools/translate.py

```diff
@@ -154,7 +154,7 @@ def encode(s):
 TRANSLATED_ATTRS = dict.fromkeys({
     'string', 'add-label', 'help', 'sum', 'avg', 'confirm', 'placeholder', 'alt', 'title', 'aria-label',
     'aria-keyshortcuts', 'aria-placeholder', 'aria-roledescription', 'aria-valuetext',
-    'value_label', 'data-tooltip', 'data-editor-message',
+    'value_label', 'data-tooltip', 'data-editor-message', 'href',
```

Comment:
because of https://github.com/odoo/odoo/blob/53c107317019f2f07f59a665204aa19ff14d1a08/odoo/tools/translate.py#L170
the `t-attf-href` will also become translatable, could you recheck if it is what you want?